template.tpl

```diff
@@ -52,6 +52,14 @@ ___TEMPLATE_PARAMETERS___
     "help": "If enabled, cookies generated by Data tag will be reseated from the server with an expiration time of two years. Useful if you use Data tag store functionality.",
     "defaultValue": true
   },
+  {
+    "type": "CHECKBOX",
+    "name": "acceptMultipleEvents",
+    "checkboxText": "Accept Multiple Events",
+    "simpleValueType": true,
+    "help": "When the Accept Multiple Events is set to true, the Data Client will accept an array of events in the request body. It will then call the runContainer for each event in the array.",
+    "defaultValue": false
+  },
```

Comment:
change text to 



When the Accept Multiple Events is set to true, the Data Client will parse an array of objects, in the request body, as separate events.
<br><br>
Example:
<br>
[<br>
{"event":"page_view"},<br>
{"event":"view_item"}<br>
]<br>